### ISSUE TITLE
[tflite] Conv2DTranspose output difference between mobile CPU and mobile GPU 

### ISSUE THREAD

**Issue by koodzi** [tensorflow/tensorflow#42649]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No
- OS Platform and Distribution: Windows 10, Android
- Mobile device: Galaxy Fold, Pixel 3
- TensorFlow installed from binary
- TensorFlow version: v2.3.0-rc2-23-gb36436b087 2.3.0
- TensorFlow Lite version  ['org.tensorflow:tensorflow-lite:0.0.0-nightly'](https://github.com/koodzi/tflite-comparing-cpu-gpu-out/commit/e9b92114cd412c3d9fbee0f4a5adf8db3e405ca6#diff-39e7d8c00954e920b98e7636f0ac30b2R46) and 2.3.0
- Python version: 3.7
- Bazel version (if compiling from source): -
- GCC/Compiler version (if compiling from source): -
- CUDA/cuDNN version: - 
- GPU model and memory: Adreno 640

**Describe the current behavior**
In a specific situation, `tf.keras.layers.Conv2DTranspose` [executed on mobile CPU and on mobile GPU](https://github.com/koodzi/tflite-comparing-cpu-gpu-out/commit/e9b92114cd412c3d9fbee0f4a5adf8db3e405ca6#diff-31e60fe4136762a27be59770ef684d01R71) returns different outputs -the mean difference between the output from CPU and output from GPU is  ~0.1 - 0.7 but the difference should be ~1e-07.
([my metric](https://github.com/koodzi/tflite-comparing-cpu-gpu-out/commit/e9b92114cd412c3d9fbee0f4a5adf8db3e405ca6#diff-31e60fe4136762a27be59770ef684d01R117) - mean difference => abs(cpu_output - gpu_output) => and it shoould be close to zero)

Specific situation:
* tf.keras.layers.Conv2DTranspose
* Input shape has odd width (NWHC) e.x. (1, 9, 6, 45)
* padding='same', kernel_size=(5, 3), strides=(2, 1),

[model](https://github.com/koodzi/tflite-comparing-cpu-gpu-out/commit/e9b92114cd412c3d9fbee0f4a5adf8db3e405ca6#diff-98d852ec33b2563b33b6c75585566a06R5)

[incorrect model - model_01](https://github.com/koodzi/tflite-comparing-cpu-gpu-out/commit/e9b92114cd412c3d9fbee0f4a5adf8db3e405ca6#diff-98d852ec33b2563b33b6c75585566a06R52)

**Describe the expected behavior**
The difference between the two outputs should be close to zero.

**Standalone code to reproduce the issue**
[https://github.com/koodzi/tflite-comparing-cpu-gpu-out](https://github.com/koodzi/tflite-comparing-cpu-gpu-out)

**Other info/logs** Include any logs or source code that would be helpful to
![diff](https://user-images.githubusercontent.com/18346395/91163679-42683b00-e6ce-11ea-8d6f-bb3128fe3a00.png)

**Comment by kumariko** [comment]
@koodzi It looks like you are using an older Version of Tensorflow. Many bugs have been fixed in the latest version. Could you please execute your code using Latest Version of TF 2.6 and let us know if the issue still persists? Thanks!

**Comment by koodzi** [comment]
@kumariko the problem still persists 
I have updated the TensorFlow and TFLite to 2.6 and created new models, but there is still difference between CPU and GPU outputs on Pixel3 and Samsung Fold1.

**Comment by pkgoogle** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFLite team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the TFLite space.
